docs/reference/client/README.md

```diff
@@ -0,0 +1,6 @@
+# Knative Client Reference
+
+Please see the `kn` documentation available at:
+
+[{{ clientdocs() }}]({{ clientdocs() }}){target=_blank}
```

Comment:
```suggestion
See the [`kn` documentation]({{ clientdocs() }}){target=_blank} in GitHub.
```

Reply:
I think it is better for accessibility purposes to have link text instead of a URL. Also it is in our style guide.

Reply:
Also, should this link go to the 1.1 branch? At the moment, from the [preview](https://deploy-preview-4623--knative.netlify.app/development/reference/client/) it looks like it's going to main.

Reply:
> Also, should this link go to the 1.1 branch? At the moment, from the [preview](https://deploy-preview-4623--knative.netlify.app/development/reference/client/) it looks like it's going to main.

It should get updated automatically depending on which branch is being built, similar to how the serving/eventing yaml links in the getting started guide are handled.

Reply:
Ah I see now! Since we're in main it'll go to main.